metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -315,6 +317,70 @@ class Compilers(
       .getOrElse(Future.successful(Nil))
   }
 
+  // sasaki dev area is start↓
+
+  def semanticTokens(
+      params: SemanticTokensParams,
+      capableTypes:List[String],
+      capableModifiers:List[String],
+      token: CancelToken
+  ): Future[SemanticTokens] = {
+    scribe.info("Debug: Compiliers.semanticTokens:Start")
+
+    // Probably extra prcess is needed to construct vFile for Scala 3.
+    // See didchange.
+    val path = params.getTextDocument.getUri.toAbsolutePath
```

Comment:
We can add a check on the path to make sure that we don't yet handle .sbt and .sc files.

Reply:
I'll add check if (filename.isScalaScript || filename.isSbt)` then return null list .